robotbase/src/main/java/com/team2357/frc2022/commands/RecordPath.java

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package com.team2357.frc2022.commands;
+
+import java.io.File;
+import java.io.PrintWriter;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+
+import com.team2357.frc2022.util.Utility;
+import com.team2357.lib.subsystems.drive.FalconTrajectoryDriveSubsystem;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class RecordPath extends CommandBase {
```

Comment:
Can we have our command classes end in "Command" please? I think this is something we've done in the past, right? And it definitely helps us remember what we're dealing with when we instantiate these and use them.

Reply:
Done